styles/style.css

```diff
@@ -70,7 +72,7 @@ p {
   padding: .5rem 1em;
 }
 .navigation__item:hover{
-  background-color: var(--orange-light);
+  background-color: var(--orange-dark);
```

Comment:
this hover effect will make the navigation look like the buttons I think it should be only changing the color, not the background color

Reply:
Yes, however, there are many websites where the links appear to have very similar styles.